app/pathogen/arbovirus/analyze/columns.tsx

```diff
@@ -60,7 +60,7 @@ export const columns: ColumnDef<Estimate>[] = [
           variant="ghost"
           onClick={() => column.toggleSorting(column.getIsSorted() === "asc")}
         >
-          Pathogen
+          Arbovirus
```

Comment:
PR description:

> I didn't bother with the plots since we're going to change those before showing the conference. The name of the property is still `pathogen` in the app, I'm just changing what it's displayed as. If we want to change the name of the property too on like the API and the rest of the front end (I'm not sure if there's a huge amount of value in that though), let's make it a tech debt ticket. I know there's new instances in PRs open right now so I'll probably just make suggestions on their PRs to use the right word as soon as this gets merged. If their stuff gets merged before mine, I'll change it myself.